docs/TryItOut.md

```diff
@@ -41,7 +41,7 @@ Then you can create an invoice, either through
 
 First choose where to setup your store.
 Either:
-* Host your store on someone else's BTCPay Server. You can find a list of third-party hosts [here](thirdpartyhosting.md/the-list-of-btcpay-third-party-hosts).
+* Host your store on someone else's BTCPay Server. You can find a list of third-party hosts [here](./ThirdPartyHosting.md/the-list-of-btcpay-third-party-hosts).
```

Comment:
We need a `#` instead of the `/` here – like this:

```
[here](./ThirdPartyHosting.md#the-list-of-btcpay-third-party-hosts)
```

Reply:
```suggestion
* Host your store on someone else's BTCPay Server. You can find a list of third-party hosts [here](./ThirdPartyHosting.md#the-list-of-btcpay-third-party-hosts).
```